stories/Containers/StoreHoursList.stories.js

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { StoreHoursList } from '../../src';
+
+const sampleCategories = [
+    {
+        category: 'Winter',
+        hoursByDay: {
+            monday: [                    
+                {
+                    from: '12:00',
+                    to: '20:00'
+                }], 
+            tuesday: [], 
+            wednesday: [], 
+            thursday: [
+                {
+                    from: '12:00',
+                    to: '20:00'
+                }
+            ], 
+            friday: [], 
+            saturday: [], 
+            sunday: [] 
+        }
+    },
+    {
+        category: 'Summer',
+        hoursByDay: {
+            monday: [                    
+                {
+                    from: '12:00',
+                    to: '20:00'
+                }], 
+            tuesday: [], 
+            wednesday: [], 
+            thursday: [
+                {
+                    from: '12:00',
+                    to: '20:00'
+                }
+            ], 
+            friday: [], 
+            saturday: [], 
+            sunday: [] 
+        } 
+    },
+    {
+        category: 'Holidays',
+        hoursByDay: {
+            monday: [                    
+                {
+                    from: '12:00',
+                    to: '20:00'
+                }], 
+            tuesday: [], 
+            wednesday: [], 
+            thursday: [
+                {
+                    from: '12:00',
+                    to: '20:00'
+                }
+            ], 
+            friday: [], 
+            saturday: [], 
+            sunday: [] 
+        }
+    }
+];
+
+const sampleActiveCategorySchedule = {
+    category: 'Winter',
+    hoursByDay: {
+        monday: [                    
+            {
+                from: '12:00',
+                to: '20:00'
+            }],
+        tuesday: [], 
+        wednesday: [], 
+        thursday: [
+            {
+                from: '12:00',
+                to: '20:00'
+            }
+        ], 
+        friday: [], 
+        saturday: [], 
+        sunday: [] 
+    }
+};
+
+storiesOf('StoreHoursList', module)
+    .add('with default', () => (
+        <StoreHoursList
+            allCategories={sampleCategories}
+            oneCategorySchedule={sampleActiveCategorySchedule}
```

Comment:
What does this prop mean versus the other prop?
Who is defining the active schedule is it the data structure or is it a prop called activeCategorySchedule?